### PR TITLE
Distinguish ${camel.version} and ${camel-community.version}

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
The decision which Camel artifact in CQ BOM gets `${camel.version}` and which `${camel-community.version}` is based on the presence of the given artifact in `org.apache.camel:camel-bom:${camel.version}` (here `${camel.version}` is the productized version). 

@cunningt do you think this approach is correct and reliable over time?

**Comment by cunningt** [comment]
I think having a separate property for community version should be okay - the only issue we might run into is if we productize a camel component that we're assigning the community version to, PME may try to align that component to the productized version.    I can try adding some checks on that.

**Comment by cunningt** [comment]
This change looks good to me.     This should work well, the only place where we would run into trouble would be if we do end up productizing a component we want a community version for - PME may try to align that.

File: pom.xml [hunk 1]
```diff
@@ -39,7 +39,8 @@
     <properties>
         <!-- Primary dependencies - maintained manually -->
         <camel.major.minor>3.11</camel.major.minor> <!-- run after each change: cd docs && mvnd validate -->
-        <camel.version>${camel.major.minor}.1</camel.version>
+        <camel-community.version>${camel.major.minor}.1</camel-community.version>
+        <camel.version>3.11.1.fuse-800009</camel.version>
```
Comment (jamesnetherton) [inline pom.xml]: Should we align the `camel-dependencies` parent version with this?

Comment (ppalaga) [reply pom.xml]: Good catch!

Comment (cunningt) [reply pom.xml]: If we use the productized version (3.11.1.fuse-800009-redhat-00001) for camel.version it won't disappear in two weeks like the dev build will - you may need to be on VPN to build though.

Comment (ppalaga) [reply pom.xml]: OK, let me use that one.

**Pull request merged**